test/report/merge.t

```diff
@@ -0,0 +1,13 @@
+Merge two files
+
+  $ echo "(lang dune 2.7)" > dune-project
+  $ cat > dune <<'EOF'
+  > (executables
+  >  (names test_merge1 test_merge2)
+  >  (instrumentation (backend bisect_ppx)))
+  > EOF
+  $ dune exec ./test_merge1.exe --instrument-with bisect_ppx
+  $ dune exec ./test_merge2.exe --instrument-with bisect_ppx
+  $ bisect-ppx-report merge merged.coverage
```

Comment:
you could also test running `bixect-ppx-report summary --per-file` (I think this would run the report by merging the files internally in bisect-ppx-report) and then running `bixect-ppx-report summary --per-file merged.coverage` and verifying that the result is indeed the same.

Reply:
That's a very good idea! I removed the `cat merged.coverage` as `bisect-ppx-report summary --per-file` is way more readable :)